pkgs/development/libraries/openssl/default.nix

```diff
@@ -154,6 +154,12 @@ let
 
     enableParallelBuilding = true;
 
+    # openssl3 runs `pod2man` as part of `make install`, which means that the
+    # installPhase takes longer than the buildPhase without this:
```

Comment:
Are you sure it's not simply reaching a previously unbuilt `doc` target or thereof in order to reach the install target?

In that case you should rather make the buildPhase explicitly build that target in addition to the regular targets via `buildFlags`.